ports/mimxrt10xx/common-hal/microcontroller/Processor.c

```diff
@@ -50,6 +53,9 @@ float common_hal_mcu_processor_get_temperature(void) {
     OCOTP_Deinit(OCOTP);
 
     return temp;
+    #else
+    return 0;
```

Comment:
I'd prefer to raise `NotImplementedErrror` here. I looked at the other ports. Espressif raises if temp is not available. STM returns `false`  (? - that is, zero), a few ports return `NaN`, which I think is kind of a bad idea. We should fix all those to raise.

Reply:
same for `get_voltage` below

Reply:
The shared-bindings are documented that they return None when not available. They convert NAN to None so NAN matches the docs.